emails/utils.py

```diff
@@ -111,6 +117,8 @@ def ses_send_raw_email(
             ConfigurationSetName=settings.AWS_SES_CONFIGSET,
         )
         incr_if_enabled("ses_send_raw_email", 1)
+        profile = Profile.objects.filter(user__email=to_address).first()
+        profile.update_abuse_metric(email_forwarded=True)
```

Comment:
suggestion (non-blocking): this adds an extra DB query to every email send operation. is there a way we can move this code to a place where we already have the `profile` object?

Reply:
Definitely possible, it just requires more code in different places that could call the `ses_send_raw_email`. I can do this if you think this should be a blocker.

Reply:
John and I looked at the code and also noted that the `update_abuse_metrics` was double counting for replies too so we decided this was a blocking change and made the update on [5a4a83a](https://github.com/mozilla/fx-private-relay/pull/2409/commits/5a4a83a8913680e99eba00fbff093bc105fdb34e)